source/classes.tex

```diff
@@ -4472,8 +4472,8 @@
 \indextext{access control!\idxcode{protected}}%
 protected;
 that is, its name can be used only by members and friends
-of the class in which it is declared, by classes derived from that class, and by their
-friends (see~\ref{class.protected}).
+of the class in which it is declared, and members of classes derived from that class
```

Comment:
One more concern here: We change "by classes" to "by members of classes". This change seems to exclude the base-specifier, which is not in a member. Example:

struct B { };
struct D1 : protected B { };
struct D2 : D1  {};

I think the injected-class-name B is protected in D1, thus accessible from D2. But the name doesn't appear in a member.

That said, this paragraph is just one of those intro sentences where the actual rules appear later. (Note the funny punctuation, btw.) So, we shouldn't say wrong things, but we don't need to be ultra-precise either.